modules/serverless-ossm-traffic-isolation-architecture.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: CONCEPT
+[id="serverless-ossm-traffic-isolation-architecture_context"]
+= High-level architecture
+
+The high-level architecture of {ServerlessProductShortName} traffic isolation provided by {SMProductShortName} consists of `AuthorizationPolicies` in the `knative-serving`, `knative-eventing` and the tenants namespaces, with all the components being part of the {SMProductShortName}. The injected {SMProductShortName} sidecars take care of enforcing those rules to isolate network traffic between tenants.
+
+[id="serverless-ossm-traffic-isolation-architecture-serving"]
+== Knative Serving
+
+The Knative Serving adheres to the following architecture:
+
+image::multi-tenancy-service-mesh-serving.png[Knative Serving architecture]
+
+The following relationships apply:
+
+. Workloads in project `tenant-2` can not call workloads in project `tenant-1` directly. Requests are rejected in the Istio proxies of `tenant-1` workloads.
+
+. Workloads in project `tenant-2` can not call workloads in project `tenant-1` through `ingress-gateway`. Requests are rejected in the Istio proxies of `tenant-1` workloads.
```

Comment:
Should we use `istio-ingressgateway` instead of `ingress-gateway`? When it's enclosed in apostrophes it should probably mean some real component. But nothing like `ingress-gateway` exists. Or we could use just "ingress gateway" so that we speak more generally.